fleetspeak/src/server/https/https_test.go

```diff
@@ -79,7 +86,28 @@ func makeServer(t *testing.T, caseName, clientCertHeader string, frontendMode cp
 	return ts.S, ts.DS, tl.Addr().String()
 }
 
-func makeClient(t *testing.T) (common.ClientID, *http.Client, []byte) {
+func clientCertFingerprint(derBytes []byte) (string) {
```

Comment:
If this is the same as the one in the other test: should we move it to a test library? (can be a followup)

Reply:
Totally agree. Since this will span both `client` and `server` would `common` be a good place to put it?
Will come and have a brief chat when I am back to get your advice.

Reply:
As discussed we leave this like it is.